src/perf/bin/drvmain.cpp

```diff
@@ -116,7 +116,9 @@ void __cdecl operator delete[] (_In_opt_ void* Mem, _In_opt_ size_t) {
     }
 }
 
+#ifndef QUIC_BUILD_STATIC
```

Comment:
This change actually isn't necessary, because we will never support kernel mode statically built. There are many more issues with doing that.